content/90.back-matter.md

```diff
@@ -6,7 +6,54 @@
 <style> span.gene_color { color:#02b3e4 } span.disease_color { color:#875442 } span.compound_color { color:#e91e63 } </style> 
 ## Supplemental Figures {.page_break_before}
 
-### Confidence Scores for Each Edge Type 
+### Top Edge Prediction Tables
```

Comment:
Table looks great. Excited to see how it renders with the coloring... could be very slick. We may consider adding a smaller version to the main text, like with 5-10 rows, but we can decide later